ibm_catalog.json

```diff
@@ -46,8 +46,8 @@
       "support_details": "This product is in the community registry, as such support is handled through the originated repo. If you experience issues please open an issue in that repository [https://github.com/terraform-ibm-modules/terraform-ibm-account-infrastructure-base/issues](https://github.com/terraform-ibm-modules/terraform-ibm-account-infrastructure-base/issues). Please note this product is not supported via the IBM Cloud Support Center.",
       "flavors": [
         {
-          "label": "Fully configurable",
-          "name": "fully-configurable",
+          "label": "Resource groups with Account Settings",
+          "name": "resource-groups-with-account-settings",
```

Comment:
This is a breaking change - if we want to avoid this we can revert this value

Reply:
maybe we can just update the `label`, and keep the same `name`. The `label` is what a user will see in the UI

Reply:
Hmm, although since we are renaming `global_resource_group_name` and all our beta tiles are using this (and not pinned to an exact DA version), this is going to be a breaking change anyway. Can I suggest that we move the DA code from `solutions/fully-configurable` -> `solutions/resource-groups-with-account-settings` since we are changing this?

Reply:
I think it might be ok, both variations use the same directory and just configure differently, so in this context I think it is fully configurable.

Reply:
agree - ignore